cmd/flow-dps-live/main.go

```diff
@@ -223,16 +218,37 @@ func run() int {
 		"0.0.0.0:0", // automatically choose port, listen on all IPs
 		seedNodes,
 		unstaked.WithBootstrapDir(flagBootstrap),
-		unstaked.WithDB(data),
+		unstaked.WithDB(protocolDB),
 		unstaked.WithLogLevel(flagLevel),
 	)
 	if err != nil {
 		log.Error().Err(err).Str("bucket", flagBucket).Msg("could not create consensus follower")
 		return failure
 	}
 
-	// Initialize the execution follower that will read block records from the
-	// Google Cloud Platform bucket.
+	// There is a problem with the Flow consensus follower API which makes it
+	// impossible to use it to bootstrap the protocol state. The consensus
+	// follower will only bootstrap it when it's starting. This makes it
+	// impossible to initialize our consensus tracker, which needs a valid
+	// protocol state, and to add it to the consensus follower for block
+	// finalization, without missing some blocks. As a work-around, we manually
+	// bootstrap the Flow protocol state using the bootstrap data here.
```

Comment:
Shouldn't we link this note to a FlowGo issue? This way when it gets fixed, we can remove this logic.

Reply:
I don't think this will be fixed, because it's really tightly integrated with the entire node logic. Would be a major refactoring.